examples/multi_piece_shipment.py

```diff
@@ -0,0 +1,130 @@
+#       https://goshippo.com/docs/reference#addresses
+address_from = {
+    "name": "Shippo Team",
+    "street1": "965 Mission St",
+    "street2": "Unit 480",
+    "city": "San Francisco",
+    "state": "CA",
+    "zip": "94103",
+    "country": "US",
+    "phone": "+1 555 341 9393",
+}
+
+# Example address_to object dict
+# The complete fence for the address object is available here:
+#       https://goshippo.com/docs/reference#addresses
+
+address_to = {
+    "name": "Shippo Friend",
+    "street1": "1092 Indian Summer Ct",
+    "city": "San Jose",
+    "state": "CA",
+    "zip": "95122",
+    "country": "US",
+    "phone": "+1 555 341 9393",
+}
+
+
+# parcel object dict
+# The complete reference for parcel object is here:
+#       https://goshippo.com/docs/reference#parcels
+parcel = {
+    "length": "3",
+    "width": "3",
+    "height": "3",
+    "distance_unit": "in",
+    "weight": "1",
+    "mass_unit": "lb",
+}
+
+# Parcels do not need to be the same!
+parcel_list = []
+for parcel_piece in range(0,10):
+    parcel_list.append(parcel)
+
+print("We now have {} in this shipment!".format(len(parcel_list)))
+carrier = [carrier_account]
+
+# Example shipment object
+# For complete reference to the shipment object:
+#      https://goshippo.com/docs/reference#shipments
+shipment = shippo.Shipment.create(
+    address_from=address_from,
+    address_to=address_to,
+    parcels=parcel_list,
+    carrier_accounts=carrier,
+    asynchronous=True
+)
+
+# Rates are stored in the `rates` array
+# The details on the returned object are here:
+#      https://goshippo.com/docs/reference#rates
+# For demo purposes enter the corresponding enumerated rate and press Enter.
+shipment_id = shipment.object_id
+print(f"the shipment_id {shipment_id}")
+rates = shippo.Shipment.get_rates(shipment.object_id)
+print(rates.results)
+for x,y in enumerate(rates.results):
+    print(f"{x}: amount: {y['amount']},rate_object_id {y['object_id']}")
+the_rate = int(input("Enter number next to rate"))
+rate = rates.results[the_rate]
+print("rate selected: \n\n {}".format(rate))
+
+# Purchase the desired rate with a transaction request
+transaction = shippo.Transaction.create(
+    rate=rate.object_id, label_file_type="PDF_A4",
+    asynchronous=True)
+
+# print the shipping label from label_url
+# Get the tracking number from tracking_number
+if transaction.status == "QUEUED":
+    print("the_return_response async {}".format(transaction))
+    #print("Purchased label with tracking number %s" %
+    #      transaction.tracking_number)
+    #print("The FIRST label can be downloaded at %s" % transaction.label_url)
+    # 
+
+else:
+    print("Failed purchasing the label due to:")
+    for message in transaction.messages:
+        print("- %s" % message['text'])
+
+print("This is async so we need to get the transaction + rate_id once queded has changed (to retrieve the remaining labels)") 
+
+TIMEOUT = 60  # thirty second timeout due to retry_sleep
```

Comment:
I'd still prefer renaming this constant to RETRY_LIMIT or something similar. timeout doesn't really describe the true usage of this constant. Also since retry_sleep is a constant, can you upper case that too?

Reply:
makes sense, done.